apps/confidential/confidential_121_plus.go

```diff
@@ -0,0 +1,19 @@
+//go:build go1.21
+
+package confidential
+
+import (
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/internal/slog"
```

Comment:
When pkgsite renders docs for `WithLogger`, it creates a link to the definition of the parameter's type i.e., `slog.Logger`. If that type is defined in your `internal/slog` package, the doc link will point there, meaning anyone who clicks that link ends up at docs for an internal package they can't reference. Of course that package references types from `logs/slog` and therefore its docs link in turn to that package, but you can avoid linking to your internal package by importing `logs/slog` here:
```suggestion
	"log/slog"
```
(this doesn't break the build because the compiler is smart enough to realize that your `internal/slog.Logger` is the same type as `logs/slog.Logger`)

Reply:
Interesting to know thanks!